Tools/LambdaTestTool/src/Amazon.Lambda.TestTool/Utils.cs

```diff
@@ -123,7 +123,7 @@ public static string FindLambdaProjectDirectory(string lambdaAssemblyDirectory)
             return FindLambdaProjectDirectory(Directory.GetParent(lambdaAssemblyDirectory)?.FullName);
         }
 
-        public static IList<string> SearchForConfigFiles(string lambdaFunctionDirectory)
+        public static IList<string> SearchForConfigFiles(string lambdaFunctionDirectory, bool noConsoleLog = false)
```

Comment:
Change name of `noConsoleLog` to `disableLogging` to match my suggest name for the switch.

Reply:
Done. Please review.